go.mod

```diff
@@ -252,7 +252,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101411.5-rc.1
+replace github.com/ethereum/go-ethereum => github.com/mdehoog/op-geth v0.0.0-20250102015754-dc3e6fe74538
```

Comment:
will update once https://github.com/ethereum-optimism/op-geth/pull/457 is merged